core/src/main/resources/lib/layout/breadcrumbs.js

```diff
@@ -49,11 +49,14 @@ var breadcrumbs = (function() {
         return a+'/'+b+qs;
     }
 
-    function postRequest(action, event, url) {
-        new Ajax.Request(url);
+    function postRequest(action, event, cfg) {
```

Comment:
any chance it's used somewhere else? I am not sure whether it may break compatibility

Reply:
not found any over usage

Reply:
@tfennelly @kzantow @recena Do you see any risks?